jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -85,6 +85,7 @@ pipeline {
             post {
                 always {
                     archiveArtifacts artifacts: 'commits.yml'
+                    postCleanup()
```

Comment:
Do we need to update the regression file since we changed this?

Reply:
No need, since we are not directly testing this jenkinsfile I believe.
Will see if the checks failed because of this.